### PR TITLE
Fix documentation for ClientSSLCertSecret and ServerSSLCertSecret

### LINKED ISSUE #839: serverSSLCertSecret: no certificates found
SSL is enabled for all listeners, used serverSSLCertSecret in CR
kafka-server is the secret under kafka namespace used to refer in serverSSLCertSecret. it has keystore.jks, truststore.jks, password
Yet it throws no certificates found\nfailed to decode certificate Error
image

### PR DESCRIPTION
**Pull request by bartam1**
| Q               | A      |
| --------------- | ------ |
| Bug fix?        | yes |
| New feature?    | no |
| API breaks?     | no |
| Deprecations?   | no |
| Related tickets | fixes #839  |
| License         | Apache 2.0 |

### What's in this PR?
It fixes the documentation for the ClientSSLCertSecret and ServerSSLCertSecret.

### Why?
The secrets which are referenced by the ClientSSLCertSecret and ServerSSLCertSecret need to contain other fields than the keystore.jks, truststore.jks and password when they are used for inner broker or controller communication

### CODE REVIEW

File: api/v1beta1/kafkacluster_types.go [hunk 1]
```diff
@@ -91,8 +91,9 @@ type KafkaClusterSpec struct {
 	// It will be used by the koperator, cruise control, cruise control metrics reporter
 	// to communicate on SSL with that internal listener which is used for interbroker communication.
 	// The client certificate must share the same chain of trust as the server certificate used by the corresponding internal listener.
-	// The secret must contains the keystore, truststore jks files and the password for them in base64 encoded format
-	// under the keystore.jks, truststore.jks, password data fields.
+	// The secret must contain the keystore, truststore jks files and the password for them in base64 encoded format and
+	// the tls certificate, tls private key, CA certificate in PEM and base64 encoded format
```
Comment (panyuenlau) [inline api/v1beta1/kafkacluster_types.go:95]:
```suggestion
	// the tls certificate, tls private key, CA certificate in PEM format with base64 encoding
```

File: api/v1beta1/kafkacluster_types.go [hunk 2]
```diff
@@ -510,7 +511,7 @@ type CommonListenerSpec struct {
 	// +kubebuilder:validation:Enum=ssl;plaintext;sasl_ssl;sasl_plaintext
 	Type SecurityProtocol `json:"type"`
 	// ServerSSLCertSecret is a reference to the Kubernetes secret that contains the server certificate for the listener to be used for SSL communication.
-	// The secret must contain the keystore, truststore jks files and the password for them in base64 encoded format
+	// The secret must contain the keystore, truststore jks files and the password for them in base64 encoded format and the tls certificate in PEM and base64 encoded format
```
Comment (panyuenlau) [inline api/v1beta1/kafkacluster_types.go:514]:
```suggestion
	// The secret must contain the keystore, truststore jks files and the password for them in base64 encoded format and the tls certificate in PEM format with base64 encoding
```

**Review by panyuenlau — DISMISSED** [review]
Not sure if these are the better way of rephrasing them

**Review by pregnor — APPROVED** [review]
LGTM aside from Darren's comments

**Review by stoader — CHANGES_REQUESTED** [review]
Run `make manifests` to ensure that all artifacts are updated

**Review by pregnor — APPROVED** [review]
LGTM

**Review by stoader — APPROVED** [review]

**Review by panyuenlau — APPROVED** [review]

**Comment by panyuenlau** [comment]
@bartam1 Can we merge this?

**Pull request merged**